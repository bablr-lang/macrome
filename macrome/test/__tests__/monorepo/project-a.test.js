const { resolve } = require('path');
const { testProject } = require('../../test-project');

describe('monorepo project a', () => {
  testProject(resolve(__dirname, '../../sandbox/projects/monorepo/project-a'));
});
