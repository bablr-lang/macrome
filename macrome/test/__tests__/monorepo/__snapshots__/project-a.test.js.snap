// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`monorepo project a cleans 1`] = `
Array [
  " D lib/generated-project-a.js",
]
`;
